source/edge-server/configure.txt

```diff
@@ -134,271 +116,160 @@ EC2, you must open ports to accept incoming connections:
-            }
-
-         With this example, we explicitly sync data from the ``Item`` and 
-         ``Project`` collections, because the App Services schema also 
-         has a ``Team`` collection that we don't want to sync to the 
-         Edge Server. 
-         
-         If we wanted to sync all the data in the App Services schema, we 
-         could use the default value for the query: ``query: "*"``. 
-         
-         We could also use more granular queries to sync a subset of
-         documents with a collection. For example, only sync Projects where 
-         ``facility_id == 42``.
-
-         For more details on what you could include in a query, refer to 
-         :ref:`Realm Query Language <rql>`.
-
-   .. step:: Install Dependencies
-
-      The Edge Server requires several dependencies, which are listed 
-      in the :file:`README.md` in the ``edge_server`` directory. Follow the 
-      instructions to install the required dependencies.
-
-   .. step:: Specify Additional Configuration Details (Optional)
-
-      You can run ``make`` to build the Edge Server. This creates a 
-      ``build/`` directory that contains an :file:`edge_config.json`
-      file. This configuration contains many additional fields you can change
-      if you need to customize Edge Server configuration details.
-
-      For example, an Edge Server running on EC2 would set the ``"baseURL"`` 
-      for applications to connect to the Edge Server to the public DNS of the 
-      EC2 instance, similar to 
-      ``"http://ec2-13-58-70-88.us-east-2.compute.amazonaws.com"``.
-
-      The default configuration values let you run an Edge Server instance
-      on local hardware, and connect to it through ``http://localhost:80``.
-
-      While Edge Server is still in Private Preview, the configuration and 
-      accepted values are expected to change. When Edge Server is available
-      for Public Preview, the configuration details are expected to be more
-      stable and documentation will be added with these details.
+         * - ``--insecure-disable-auth``
+           - Boolean
+           - If ``true``, users can connect anonymously to the instance through
+             Wire Protocol or Atlas Device SDK clients. Defaults to ``false``.
+
+         * - ``--no-user``
+           - Boolean
+           - If ``true``, Edge Server does not prompt for user creation. If
+             ``false``, Edge Server connects to Atlas to check if an 
+             :ref:`email/password user <email-password-authentication>` exists
+             in the service. If a corresponding email/password user does not 
+             exist, it interactively prompts for a username and password to
+             create a user. Defaults to ``false``.
+
+         * - ``--disable-ui``
+           - Boolean
+           - If true, the Edge Server instance does not run a UI on the 
+             registered ``http-port``. Defaults to ``false``.
+
+         * - ``--base-url``
```

Comment:
I think we can remove this since this isn't something that customers should be using and is meant only for internal development. I believe customers can only access `https://services.cloud.mongodb.com` and there is no other URL such as QA, dev, etc.